.github/workflows/code-style.yml

```diff
@@ -7,44 +7,33 @@ on:
     branches:
       - main
   schedule:
-    - cron: "0 0 * * *"
+    - cron: '0 0 * * *'
 
 jobs:
   php-cs-fixer:
-    name: Code style with PHP ${{ matrix.php-versions }} - ${{ matrix.composer-dependencies }} dependencies
-    runs-on: 'ubuntu-latest'
+    name: PHP CS Fixer
+    runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
       matrix:
-        php-versions: ['8.0', '8.1', '8.2']
-        composer-dependencies: ['highest', 'lowest']
+        php-versions: ['8.1']
+        composer-dependencies: ['highest']
```

Comment:
IMHO, we should target `8.0` and `lowest` to ensure that the code is "ready" for the majority of users, WDYT?

Reply:
I don't see the benefits of testing that the style is valid against the highest deps, same for Rector, the API should not be broken or if it is, the PHPStan pipeline should fail before, maybe I'm missing something? 🤔

Reply:
I don't see the point of testing code style or Rector against already tested code in CI every day, is there any?

Testing against highest deps means you can quickly see what could break in the future, while always testing against lowest deps should not change anything.

About PHPStan, I'll put back multiple versions of PHP & deps

Reply:
Oh, that's the point I was missing, I better understand your point, seems good to me then 🙂